lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+# Creates a menu of takeaway items and displays menu
+class Menu
+  attr_reader :display , :items
```

Comment:
Right now, these readers aren't doing anything for you - you're setting them up but don't have any instance variables with matching names, and then defining some methods with the same names so you're overwriting them anyway.